src/Integrations/Greenhouse/SettingsGreenhouse.php

```diff
@@ -337,119 +250,114 @@ function ($option) use ($formId) {
-						'inputType' => 'password',
-						'inputIsRequired' => true,
-						'inputValue' => !empty($apiKey) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_GREENHOUSE_API_KEY_KEY),
-						'inputIsDisabled' => !empty($apiKey),
-					],
-					[
-						'component' => 'input',
-						'inputName' => $this->getSettingsName(self::SETTINGS_GREENHOUSE_BOARD_TOKEN_KEY),
-						'inputId' => $this->getSettingsName(self::SETTINGS_GREENHOUSE_BOARD_TOKEN_KEY),
-						'inputFieldLabel' => \__('Job Board name', 'eightshift-forms'),
-						'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
-						'inputType' => 'text',
-						'inputIsRequired' => true,
-						'inputValue' => !empty($boardToken) ? $boardToken : $this->getOptionValue(self::SETTINGS_GREENHOUSE_BOARD_TOKEN_KEY),
-						'inputIsDisabled' => !empty($boardToken),
-					],
+		return [
+			$this->getIntroOutput(self::SETTINGS_TYPE_KEY),
+			[
+				'component' => 'tabs',
+				'tabsContent' => [
 					[
-						'component' => 'divider',
+						'component' => 'tab',
+						'tabLabel' => \__('API', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'input',
+								'inputName' => $this->getSettingsName(self::SETTINGS_GREENHOUSE_API_KEY_KEY),
+								'inputId' => $this->getSettingsName(self::SETTINGS_GREENHOUSE_API_KEY_KEY),
+								'inputFieldLabel' => \__('API key', 'eightshift-forms'),
+								'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
+								'inputType' => 'password',
+								'inputIsRequired' => true,
+								'inputValue' => !empty($apiKey) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_GREENHOUSE_API_KEY_KEY),
+								'inputIsDisabled' => !empty($apiKey),
+							],
+							[
+								'component' => 'input',
+								'inputName' => $this->getSettingsName(self::SETTINGS_GREENHOUSE_BOARD_TOKEN_KEY),
+								'inputId' => $this->getSettingsName(self::SETTINGS_GREENHOUSE_BOARD_TOKEN_KEY),
+								'inputFieldLabel' => \__('Job Board name', 'eightshift-forms'),
+								'inputFieldHelp' => \__('Can also be provided via a global variable.', 'eightshift-forms'),
+								'inputType' => 'text',
+								'inputIsRequired' => true,
+								'inputValue' => !empty($boardToken) ? $boardToken : $this->getOptionValue(self::SETTINGS_GREENHOUSE_BOARD_TOKEN_KEY),
+								'inputIsDisabled' => !empty($boardToken),
+							],
+						],
 					],
 					[
-						'component' => 'intro',
-						'introTitle' => \__('Options', 'eightshift-forms'),
-						'introSubtitle' => \__('Here you can find some of options specific to Greenhouse integration.', 'eightshift-forms'),
-						'introTitleSize' => 'medium',
+						'component' => 'tab',
+						'tabLabel' => \__('Options', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'intro',
+								'introSubtitle' => \__('Here you can find some of options specific to Greenhouse integration.', 'eightshift-forms'),
```

Comment:
```suggestion
								'introSubtitle' => \__('General options.', 'eightshift-forms'),
```

You're inside the Greenhouse 'page', so there is no need for repeating the same thing multiple times.